linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading/rule.yml

```diff
@@ -8,7 +8,11 @@ description: |-
     To capture kernel module loading and unloading events, use following lines, setting ARCH to
     either b32 for 32-bit system, or having two lines for both b32 and b64 in case your system is 64-bit:
     <pre>
+        {{% if 'ol' in product or 'rhel' in product %}}
```

Comment:
Please double check the alignment on this line.

Reply:
Good catch, it added some unnecessary blank lines. I fixed it.